event_handler.go

```diff
@@ -282,6 +282,15 @@ func eventHandler(ctx *IrcContext, rtm *slack.RTM) {
 		case *slack.MessageEvent:
 			// https://api.slack.com/events/message
 			message := ev.Msg
+			if !ctx.usingLegacyToken && message.User == ctx.User.ID && message.ClientMsgID == "" {
```

Comment:
There is a check with similar purpose in [print_message function](https://github.com/insomniacslk/irc-slack/blob/master/event_handler.go#L255). Might be good to consider having both checks in the same place?

Reply:
that's the code I was looking for and couldn't find, thank you! Yeah I'll move this logic there

Reply:
fixed

Reply:
not yet working, probably need to remove the bot ID check

Reply:
ok, this seems to be working now